vent/README.md

```diff
@@ -0,0 +1,15 @@
+# Vent Board 
+
+The purpose of the vent board is to open and close the valve that allows for nitrous oxide to flow into the rocket before launch. 
+
+The vent itself is used to safely remove nitrous oxide gas from the rocket tank and also functions as a way for gases to escape from the rocket while the tank is being fueled. 
+
```

Comment:
Similar to injector: external connections: CAN, external battery, linear actuator. The potentiometer feedback of the linear actuator is used to read & control its position. A pair of DACs and comparators are used to limit its motion.